### DISCUSSION TITLE
How I use plotMarkerHeatmap function to get the heatmap of genes from called peaks?

### DISCUSSION THREAD

**Discussion by MaryGoAround** [GreenleafLab/ArchR#762] [Questions / Documentation]
Hello

I am trying to plot peaks called by macs2 but likely in the rownames of markersPeaks there is no gene name to plot and all are numbers

```
heatmapPeaks <- plotMarkerHeatmap(
  seMarker = markersPeaks, labelMarkers = rownames(markersPeaks),
  cutOff = "FDR <= 0.1 & Log2FC >= 0.5",
  transpose = TRUE
)
```
`[ArchR-plotMarkerHeatmap-736c57d5033a-Date-2021-05-17_Time-12-41-55.log](https://github.com/GreenleafLab/ArchR/files/6493575/ArchR-plotMarkerHeatmap-736c57d5033a-Date-2021-05-17_Time-12-41-55.log)`

`draw(heatmapPeaks, heatmap_legend_side = "bot", annotation_legend_side = "bot") Error in (function (classes, fdef, mtable)  :    unable to find an inherited method for function ‘draw’ for signature ‘"list"’`

I have attached my log file please can you have a look 

Can anyone help me to know what I should add/adjust to get a heatmap by this function?

**Comment by rcorces** [comment]
Sorry for not answering this sooner but this issue came up with another user and we were able to identify the problem.
See https://github.com/GreenleafLab/ArchR/issues/773
